### PR TITLE
OSDOCS-5995: Adding Network Observability to the NW Operators Overview

### PR DESCRIPTION
**Pull request by skrthomas**
Version(s):

4.10+
Issue:

https://issues.redhat.com/browse/OSDOCS-5995
Link to docs preview:

https://59621--docspreview.netlify.app/openshift-enterprise/latest/networking/networking-operators-overview.html#network-observability-operator-overview-operator
QE review:
- [X] QE has approved this change.

Additional information:

### CODE REVIEW

File: networking/network_observability/network-observability-overview.adoc [hunk 1]
```diff
@@ -6,7 +6,7 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-Red Hat offers cluster administrators the Network Observability Operator to observe the network traffic for {product-title} clusters. The Network Observability uses the eBPF technology to create network flows. The network flows are then enriched with {product-title} information and stored in Loki. You can view and analyze the stored network flows information in the {product-title} console for further insight and troubleshooting.
+Red Hat offers cluster administrators the Network Observability Operator to observe the network traffic for {product-title} clusters. The Network Observability Operator uses the eBPF technology to create network flows. The network flows are then enriched with {product-title} information and stored in Loki. You can view and analyze the stored network flows information in the {product-title} console for further insight and troubleshooting.
```
Comment (skrthomas) [inline networking/network_observability/network-observability-overview.adoc:9]: I noticed that this Overview was missing "Operator" so I added it here while I'm doing some Overview-y things.

File: networking/networking-operators-overview.adoc [hunk 2]
```diff
@@ -26,4 +26,8 @@ The External DNS Operator deploys and manages ExternalDNS to provide the name re
...
\ No newline at end of file
+The Ingress Node Firewall Operator uses an extended Berkley Packet Filter (eBPF) and eXpress Data Path (XDP) plugin to process node firewall rules, update statistics and generate events for dropped traffic. The operator manages ingress node firewall resources, verifies firewall configuration, does not allow incorrectly configured rules that can prevent cluster access, and loads ingress node firewall XDP programs to the selected interfaces in the rule's object(s). For more information, see xref:../networking/ingress-node-firewall-operator.adoc#ingress-node-firewall-operator[Understanding the Ingress Node Firewall Operator]
```
Comment (skrthomas) [inline networking/networking-operators-overview.adoc:29]: This was there. I guess my changes are somehow making Git think that I added this? Not sure why git would think that.

File: networking/networking-operators-overview.adoc [hunk 3]
```diff
@@ -26,4 +26,8 @@ The External DNS Operator deploys and manages ExternalDNS to provide the name re
...
+
+[id="network-observability-operator-overview-operator"]
```
Comment (skrthomas) [inline networking/networking-operators-overview.adoc:31]: New changes

File: networking/networking-operators-overview.adoc [hunk 4]
```diff
@@ -26,4 +26,8 @@ The External DNS Operator deploys and manages ExternalDNS to provide the name re
...
+== Network Observability Operator
+The Network Observability Operator is an optional, installable Operator that allows cluster administrators to observe the network traffic for {product-title} clusters. The Network Observability Operator uses the eBPF technology to create network flows. The network flows are then enriched with {product-title} information and stored in Loki. You can view and analyze the stored network flows information in the {product-title} console for further insight and troubleshooting. For more information, see xref:../networking/network_observability/network-observability-overview.adoc#dependency-network-observability[About Network Observability Operator].
```
Comment (nathan-weinberg) [inline networking/networking-operators-overview.adoc:33]: "installable" is probably not necessary here

Comment (skrthomas) [reply networking/networking-operators-overview.adoc:33]: @nathan-weinberg I was thinking "installable" may be an important distinction from the rest of the Operators listed here, which I think are just out-of-the-box.

Comment (nathan-weinberg) [reply networking/networking-operators-overview.adoc:33]: @skrthomas totally just my opinion here - I would say "optional" fits the bill here - every operator is "installable", i.e. we don't have any operators that are "not installable"

File: networking/networking-operators-overview.adoc (hunk 2 above)
Comment (snarayan-redhat) [reply networking/networking-operators-overview.adoc:29]: this is so weird!

**Review by snarayan-redhat — COMMENTED** [review]
Looks good!

File: networking/networking-operators-overview.adoc (hunk 2 above)
Comment (jotak) [reply networking/networking-operators-overview.adoc:29]: I'm pretty sure that's just because you added a new line at the end of this line :-)

**Review by jotak — APPROVED** [review]
lgtm!

**Pull request merged**